h/static/scripts/components/GroupModeration.tsx

```diff
@@ -141,30 +145,33 @@ function AnnotationListContent({
       onAnnotationReloaded,
     ],
   );
-
-  if (annotations && annotations.length === 0) {
-    return (
-      <div
-        className="border rounded p-2 text-center"
-        data-testid="annotations-fallback-message"
-      >
-        {!filterStatus && 'There are no annotations in this group.'}
-        {filterStatus && (
-          <>
-            There are no{' '}
-            <span className="lowercase">
-              {moderationStatusInfo[filterStatus].label}
-            </span>{' '}
-            annotations in this group.
-          </>
-        )}
-      </div>
-    );
-  }
+  const fallbackMessage = useMemo(
+    () =>
+      annotations &&
+      visibleAnnotations === 0 && (
+        <div
+          className="border rounded p-2 text-center"
+          data-testid="annotations-fallback-message"
+        >
+          {!filterStatus && 'There are no annotations in this group.'}
+          {filterStatus && (
+            <>
+              There are no{' '}
+              <span className="lowercase">
+                {moderationStatusInfo[filterStatus].label}
+              </span>{' '}
+              annotations in this group.
+            </>
+          )}
+        </div>
```

Comment:
This design leaves a large blank space in the middle of the form, underneath the message. One way to reduce this would be to add more vertical padding inside the message. Maybe worth having a look at blank states in some other apps to see if there are good patterns we can borrow.